app/assets/javascripts/discourse/controllers/keyboard-shortcuts-help.js

```diff
@@ -51,6 +52,11 @@ export default Controller.extend(ModalFunctionality, {
     this.set("modal.modalClass", "keyboard-shortcuts-modal");
   },
 
+  @discourseComputed()
+  showBookmarkShortcuts() {
+    return this.siteSettings.enable_bookmarks_with_reminders;
+  },
+
   shortcuts: {
```

Comment:
I know you just hooked into the existing here, but could we set shortcuts from onOpen, and ensure it's nullified onClose I don't feel great about this gigantic hash set on the controller

Reply:
I will open another PR for this and `@` you when it is ready

Reply:
https://github.com/discourse/discourse/pull/9355